EventGenerator/src/Pileup_module.cc

```diff
@@ -133,7 +157,7 @@ namespace mu2e {
       // decay or capture time for this muon, should
       // be the same for all its daughters
       const double time = mustop->endGlobalTime() + randExp_.fire(muonLifeTime_);
-
+      std::cout<<mustop->endGlobalTime() <<" "<< randExp_.fire(muonLifeTime_)<<std::endl;
```

Comment:
I still see the histograms in this module.  And this debug printout is unprotected and so will  fill up the log files.  Please revert PileupModule to its former state: check it out from the last commit b02c8b9e8ce5cb880040c1c0f71191658a91726f, commit and push.